client/app/helpers/optionset.js

```diff
@@ -19,6 +19,9 @@ import {
 import {
   DCPHASPROJECTCHANGEDSINCESUBMISSIONOFTHEPAS_OPTIONSET,
   DCPCONSTRUCTIONPHASING_OPTIONSET,
+  DCPEXISTINGCONDITIONS_OPTIONSET,
```

Comment:
I don't see these properties on the rwcds-form model. Can you clarify where these will be used/for what part of the wrieframes?

Reply:
`dcpIsrwcdsscenario` (line 45) and `dcpExistingconditions` (line 97) are both on the rwcds model. If you look at show.hbs, it's line 159 and 194.

Reply:
Not sure why I couldn't find them earlier on the model. Thanks!!